bin/daemon.js

```diff
@@ -56,6 +56,7 @@ async function main () {
   })
 
   const registry = Prometheus.register
+  Prometheus.collectDefaultMetrics({ register: registry })
```

Comment:
Should we use the prefix option to set a prefix?
Other metrics start with `kubernetes_external_secrets_<some_metric_name>`

Reply:
I would prefer using the default metric names. This way you can have a generic dashboard for nodejs apps like the one above.

Reply:
But wouldn't it conflict if you have other services exposing the same metrics? I'm a bit out of the metrics loop so maybe its a non issue 😄 I just know we got PRs before adding the prefix to follow Prometheus best practices

Reply:
No, it wouldn't conflict, each instance gets its own set of labels.